app-ng/src/config.lisp

```diff
@@ -0,0 +1,129 @@
+(in-package #:qvm-app-ng)
+
+(global-vars:define-global-var *config*
+    (make-hash-table :test #'equalp)
+  "A hash-table holding global configuration values. See *OPTION-SPEC* for a list of those options.")
+
+;; There are two sources of configuration. Config variables are
+;; defined in order of
+;;
+;;   1. A configuration file; e.g. ~/.qvm_config
+;;   2. Command-line options; e.g. --memory-limit
+;;
+;; where variables configured in 2 may override those set in 1.
+
+;; The variable *OPTION-SPEC* below defines permissible options. The
+;; config file is a plist whose keys are the long names in
```

Comment:
You say plist here, but `load-config-file` below treats as an alist (calling `assoc`  to lookup options).